project_task_stage_change_restriction/__manifest__.py

```
@@ -0,0 +1,24 @@
+    "license": "AGPL-3",
+    'depends': ['project'],
+    'data': [
+        'security/ir.model.access.csv',
+        'views/project_task_stage_views.xml',
+    ],
+    'demo': [
+        'data/demo_project_task_stage.xml',
+    ],
+    'installable': True,
+    'application': False,
+}
```

Pylint (3.3.7)

[warning] Statement seems to have no effect

(W0104)